src/fastga/models/weight/mass_breakdown/b_propulsion/b1_engine_weight.py

```diff
@@ -47,7 +47,7 @@ def setup(self):
         self._engine_wrapper.setup(self)
 
         self.add_input(
-            "settings:weight:propulsion:k_b1", val=np.nan, desc="Engine weight tunning factor"
+            "settings:weight:propulsion:engine:k_factor", val=1.0
```

Comment:
Actually for mass k_factors, it's data and not settings in FAST-OAD_CS23 ^^' Though for mission and fuel consumtion it is a setting, may need to open an issue for that

Reply:
Let's mark them as settings, less work and closer to CS25, win/win